docs/CLI_Reference/index.md

```diff
@@ -0,0 +1,53 @@
+# Use the ORAS command line
+
+To list available commands, either run `oras` with no parameters or execute `oras help`:
+
+```
+$ oras
+Usage:
+  oras [command]
+
+Available Commands:
+  attach      Attach files to an existing artifact
+  blob        Blob operations
+  completion  Generate the autocompletion script for the specified shell
+  cp          Copy artifacts from one target to another
+  discover    Discover referrers of a manifest in the remote registry
+  help        Help about any command
+  login       Log in to a remote registry
+  logout      Log out from a remote registry
+  manifest    Manifest operations
+  pull        Pull files from remote registry
+  push        Push files to remote registry
+  repo        Repository operations
+  tag         Tag a manifest in the remote registry
+  version     Show the oras version information
+
+Flags:
+  -h, --help   help for oras
+
+Use "oras [command] --help" for more information about a command.
+```
+
+## See also
```

Comment:
Hyperlink target name is not right in this section, e.g. `oras_attach.md` should be `0_oras_attach.md`